src/librustdoc/html/static/rustdoc.css

```diff
@@ -1326,4 +1328,4 @@ kbd {
 }
 #all-types > p {
 	margin: 5px 0;
-}
\ No newline at end of file
+}
```

Comment:
Please remove this backline. :)

Reply:
Force-pushed, but i should mention that vim will keep automatically putting that in there.